python/upload_with_authentication.py

```diff
@@ -72,10 +72,18 @@
     except KeyError:
         print("You are missing one of the environment variables MAPILLARY_USERNAME, MAPILLARY_PERMISSION_HASH or MAPILLARY_SIGNATURE_HASH. These are required.")
         sys.exit()
-
-    # generate a sequence UUID
-    sequence_id = uuid.uuid4()
-
+    sequence_file=os.path.join(path,'SEQUENCE')
+    if os.path.isfile(sequence_file):
+        # read previous sequence UUID from the file
+        print('--- SEQUENCE FILE FOUND, RESUMING UPLOAD ---')
+        with open(sequence_file,'r') as f:
```

Comment:
do you need to close the file? Honestly I don't know enough of how python works, just checking.